src/import/ImportData.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace taskforce\import;
+
+use taskforce\exeptions\BadFileExeption;
+use taskforce\import\ConstructorUsers;
+
+/**
+ * Класс ImportData
+ * Отвечает за импорт данных их scv файлов
+ * 
+ */
+
+class ImportData
```

Comment:
```suggestion
final class ImportData
```
Делай классы final, если они не наследуются